canonical-websites/www.ubuntu.com
# Correct live date for Japanese service description

## Done

The date stamp of the archived file was wrong. Now corrected.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja/2017-02-09](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja/2017-02-09)
- See that the page exists
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja/2017-02-09](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja)
- See that the right-hand sidebar link is now dated 2017-02-09 and that the link works
